querybook/webapp/ui/CopyPasteModal/CopyPasteModal.tsx

```diff
@@ -10,29 +12,37 @@ interface IProps {
     displayText?: boolean;
     title?: string;
     onHide: () => any;
+    large?: boolean;
```

Comment:
i dont think large is needed as prop? since we can figure that out automatically based on how long the text is. Something like

```ts
const large = displayText && text.length > 400;
```